packages/client/hmi-client/src/components/model/petrinet/tera-initial-table.vue

```diff
@@ -110,6 +120,12 @@ const initialList = computed<
 		.filter(({ baseInitial }) => baseInitial.toLowerCase().includes(filterText.value.toLowerCase()));
 });
 
+const hasEmptyExpressions = computed(() => ({ baseInitial }) => {
+	const semanticsForThisInitial = props.modelConfiguration.initialSemanticList.filter((s) =>
+		s.target.startsWith(`${baseInitial}_`)
+	);
+	return semanticsForThisInitial.some((s) => !s.expression);
+});
```

Comment:
What do these petrinet component changes (tera-parameter-entry, tera-initial-entry, ... etc) have to do with compare-dataset? Are these fixing something else?

Reply:
false alarm, branch is out of sync, I updated it